### PR TITLE
TE-648: Enabling step 3

### PR DESCRIPTION
**Pull request by ostrizhny**
- Developer(s): @ostrizhny

- Ticket: https://spryker.atlassian.net/browse/TE-648

- Academy PR: -

- Release Group: https://release.spryker.com/release-groups/view/1117

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Silexphp              | minor         | Container             |

#### Release Notes

With this release we remove the dependency to `pimple/pimple` and replace it with  `spryker/container`. The `spryker/container` introduces some sort of mock for Pimple to not break any code which makes use of the removed Pimple class. 

 Also removed deprecated method `Silex\Application::get()`. To be able to add controller use `$app->get('controllers')->get()` instead.

-----------------------------------------

##### Change log

Initial Release

- Container `pimple/pimple` has been replaced by `spryker/container`.
- Removed deprecated method  `Silex\Application::get()`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1",
-        "pimple/pimple": "^1.0",
+        "spryker/container": "^1.0",
```
Comment (gechetspr) [inline composer.json]: Shouldn't this be `1.0.0`?

Comment (stereomon) [reply composer.json]: It must be `^1.1.0` as in the Pimple mock is only available with the next release of the `spryker/container` module.

File: composer.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1",
-        "pimple/pimple": "^1.0",
```
Comment (ehsanmx) [inline composer.json]: removing dependency is not minor

Comment (stereomon) [reply composer.json]: In this case, it's a replacement and will still have Pimple available

**Pull request merged**